### PR TITLE
update alerting section

### PR DESCRIPTION
**Pull request by rasheedamir**

### CODE REVIEW

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-187-4e4c704a`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-187-6f01218f`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-187-8617d4cd`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-187-b834c7b3`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-187-35045678`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-187-9e657ccf`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-187-891407a6`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-187-d22b060a`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-187-f12e5751`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-187-f6c23b43`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-187-b4097557`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-187-be687b6f`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-187-f65d3be7`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-187-2b3d1f75`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-187-586a4cea`

File: content/sre/alerting/downtime-notifications-uptimerobot.md [hunk 1]
```diff
@@ -41,4 +45,9 @@ endpointMonitor:
 
 ## 5. Validate downtime notification
 
-From OpenShift console change replicas to zero; and you should recieve downtime notification!
+Reduce replicas to zero; and you should recieve downtime notification!
```
Comment (hanzala1234) [inline content/sre/alerting/downtime-notifications-uptimerobot.md:48]: shouldn't we test it by reducing the number of replicas in openshift console live?

Comment (hanzala1234) [reply content/sre/alerting/downtime-notifications-uptimerobot.md:48]: if we have to do it in gitops way, then there should be section to increase number of replicas back

File: content/sre/alerting/log-alerts.md [hunk 2]
```diff
@@ -24,7 +26,10 @@ The configuration to parse/match/send logs can be specified in the [Application
 |.Values.deployment.fluentdConfigAnnotations.key|specify log field to match the regex|
 |.Values.deployment.fluentdConfigAnnotations.pattern|specify regex to be matched|
```
Comment (hanzala1234) [inline content/sre/alerting/log-alerts.md:28]: should we add example here?

Comment (hanzala1234) [reply content/sre/alerting/log-alerts.md:28]: we also need to add slack field here

Comment (hanzala1234) [inline content/sre/alerting/log-alerts.md:28]: we need to add slack field here as well

File: content/sre/onboarding/tenant-onboarding.md [hunk 3]
```diff
@@ -1,10 +1,27 @@
...
-Create new folder at the root of repository with tenant name then create config folder for that tenant. Inside config folder add environment folder for **each** environment.
+We broadly categorize tenants into two types:
 
-Create following folders:
+1. application tenants: tenants who build and delivery applications
+2. sre|delivery-engineering|devops tenant: team responsible to onboard other tenants; usually we recomend only one such tenant
+
+Adding new tenant can be divided into parts
+
+1. Create folders with `.gitignore` files
```
Comment (hanzala1234) [inline content/sre/onboarding/tenant-onboarding.md]: you mean .gitkeep?

File: content/sre/alerting/downtime-notifications-uptimerobot.md (hunk 1 above)
Comment (rasheedamir) [reply content/sre/alerting/downtime-notifications-uptimerobot.md:48]: if you do via console then they are increased automatically by gitops

File: content/sre/alerting/log-alerts.md (hunk 2 above)
Comment (rasheedamir) [reply content/sre/alerting/log-alerts.md:28]: plz add

File: content/sre/onboarding/tenant-onboarding.md (hunk 3 above)
Comment (rasheedamir) [reply content/sre/onboarding/tenant-onboarding.md]: yeah good catch

File: content/sre/alerting/downtime-notifications-uptimerobot.md (hunk 1 above)
Comment (hanzala1234) [reply content/sre/alerting/downtime-notifications-uptimerobot.md:48]: but you will get notifications even its down for some seconds. Doing it via console is easiest way

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-187-6fa1e5d6`

File: content/sre/alerting/downtime-notifications-uptimerobot.md (hunk 1 above)
Comment (rasheedamir) [reply content/sre/alerting/downtime-notifications-uptimerobot.md:48]: as discussed gitops brings it back if done from console

**Review by hanzala1234 — APPROVED** [review]

**Comment by stakater-user** [comment]
@hanzala1234 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.130`

**Pull request merged**